api/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+const contactController = require("../controllers/contacts");
+
+router.get("/", contactController.getAll);
+router.get("/:contactId", contactController.getById);
+router.post("", contactController.addContact);
```

Comment:
```suggestion
router.post("/", contactController.addContact);
```
Trzeba przy post dodać `/`